internal/pkg/daemon/seccompprofile/seccompprofile.go

```diff
@@ -113,6 +146,11 @@ type Reconciler struct {
 
 // Reconcile reconciles a SeccompProfile.
 func (r *Reconciler) Reconcile(_ context.Context, req reconcile.Request) (reconcile.Result, error) {
+	// Mark the controller as ready if the first reconcile has been finished
```

Comment:
So the controller is considered fully ready even if only one daemon has finished?

Reply:
Right now, the daemonset pod is considered to be ready if the initial seccomp profile reconcile has been finished. We should find better readiness metrics for selinux and the recorder controller somehow, but I was not sure which ones we could use.

Reply:
I was more curious about the pods running on different nodes. If the daemonset is running 1 pod on 10 nodes, and only half of them complete a Reconcile loop and the other half fail for some reason, is the controller considered ready? Or does it not matter for this?

Reply:
So, for now the daemonset only runs 1 pod for each node, otherwise we would risk reconciliation races per node. I think this will also not change in the near future if I'm not missing anything. We also only apply the readiness probes to the SPOD (daemonset instances) and not the manager, which would be a bit more complicated from my point of view.

Reply:
Right, I definitely didn't mean we should run more than one pod per node. I think I understand now, thanks.